packages/nns/README.md

```diff
@@ -1,10 +1,39 @@
 # nns-js
 
-A library for interfacing with the ledger and governance canisters of the Internet Computer's Network Nervous System.
+A library for interfacing with the Internet Computer's Network Nervous System.
 
-This library is still in active development, and new features will incrementally be available.
+[![npm version](https://img.shields.io/npm/v/@dfinity/nns.svg?logo=npm)](https://www.npmjs.com/package/@dfinity/nns) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-## Example Usage
+## Table of contents
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Features](#features)
+  - [AccountIdentifier](#factory-accountidentifier)
```

Comment:
These are also manual, right?

Reply:
Yes manually. We could potentially generate these automatically - we have the metadata - but not sure if we can have playholder within the table of contents 🤔. Maybe something I/we can try in another iteration, good idea.